src/sentry/api/endpoints/organization_integration_repos.py

```diff
@@ -44,6 +45,11 @@ def get(
             context = {"repos": []}
             return self.respond(context)
 
+        installed_repos = Repository.objects.filter(integration_id=integration.id).exclude(
+            status=ObjectStatus.HIDDEN
+        )
+        external_names = {hidden_repo.name for hidden_repo in installed_repos}
+
         install = integration_service.get_installation(
             integration=integration, organization_id=organization.id
         )
```

Comment:
on line 63 (not changed in this PR but still in here) we use a camelCase variable name, is there a reason for that?